### PR TITLE
feat: LSDV-4583-2: Image list pagination

### PR DESCRIPTION
**Pull request by nicholasrq**
### PR fulfills these requirements
- [X] Commit message(s) and PR title follows the format `[fix|feat|ci|chore|doc]: TICKET-ID: Short description of change made` ex. `fix: DEV-XXXX: Removed inconsistent code usage causing intermittent errors`
- [x] Tests for the changes have been added/updated (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
_(check all that apply)_
- [ ] Product design
- [x] Frontend

### Describe the reason for change
LSDV-4583
LSDV-4583-2

#### What is the new behavior?
Along with `valueList` support this PR brings pagination support to travel between images in the `valueList`

#### What is the current behavior?
Image with `valueList` always displays only the first image in the list

#### What libraries were added/updated?
None

#### Does this change affect performance?
No

#### Does this change affect security?
No

#### What alternative approaches were there?
None

#### What feature flags were used to cover this change?
`feat_front_lsdv_4583_multi_image_segmentation_short`

### Does this PR introduce a breaking change?
_(check only one)_
- [ ] Yes, and covered entirely by feature flag(s)
- [ ] Yes, and covered partially by feature flag(s)
- [x] No
- [ ] Not sure (briefly explain the situation below)

### What level of testing was included in the change?
_(check all that apply)_
- [x] e2e
- [ ] integration
- [ ] unit

### Which logical domain(s) does this change affect?
Image segmentation, Labeling Interface

### CODE REVIEW

File: src/common/Pagination/Pagination.tsx [hunk 1]
```diff
@@ -48,7 +54,7 @@ export const Pagination: FC<PaginationProps> = forwardRef<any, PaginationProps>(
   };
 
   return (
-    <Block name="pagination" mod={{ size }}>
+    <Block name="pagination" mod={{ size, outline, align, noPadding }}>
```
Comment (hlomzik) [inline src/common/Pagination/Pagination.tsx:57]: looks like too much parameters for tag used just twice. if the problem is in different designs maybe it's better to get them in sync?

File: src/tags/object/Image.js [hunk 2]
```diff
@@ -289,11 +291,20 @@ const Model = types.model({
   },
 
   get parsedValue() {
-    return parseValue(self.value, self.store.task.dataObj);
+    return self._parsedValue = self._parsedValue ?? parseValue(self.value, self.store.task.dataObj);
```
Comment (hlomzik) [inline src/tags/object/Image.js:294]: why do you need this temporary private vars here and in `parsedValueList`?

**Review by hlomzik — CHANGES_REQUESTED** [review]
and comments from https://github.com/heartexlabs/label-studio-frontend/pull/1173

File: src/tags/object/Image.js (hunk 2 above)
Comment (nicholasrq) [reply src/tags/object/Image.js:294]: those are not temp, it's cache

File: src/common/Pagination/Pagination.tsx (hunk 1 above)
Comment (nicholasrq) [reply src/common/Pagination/Pagination.tsx:57]: i prefer to be explicit. i could of course use single param like "bare=true" or somethig that combines these three. yet, it won't describe itself and I'll have to add documentation. also, these parameters might be useful in other pagination cases

File: src/tags/object/Image.js (hunk 2 above)
Comment (hlomzik) [reply src/tags/object/Image.js:294]: but that's the getter, so it should not be called randomly. and if `value` is changed for some reason (LSF re-init from DM for example) it should be recalculated.

Comment (nicholasrq) [reply src/tags/object/Image.js:294]: yeah, you're right

**Review by yyassi-heartex — APPROVED** [review]

File: examples/image_list/START.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (hlomzik) [inline examples/image_list/START.md:1]: weird file, remove it please

**Review by hlomzik — APPROVED** [review]

**Pull request closed without merging**